### DISCUSSION TITLE
Unable to receive mails. Getting 530 Authentication error

### DISCUSSION THREAD

**Discussion by ankitjaggi** [postalserver/postal#2507] [Help with using Postal] [answered]
I'm trying to receive emails on my Postal setup which is running on an Amazon Linux 2 machine. I have setup the required DNS configuration.

<img width="811" alt="image" src="https://github.com/postalserver/postal/assets/6972850/afa1755d-52e3-4fd2-b2e8-11513ed7b8bd">

**Comment by willpower232** [comment, marked as answer]
Is mail.jaggi.dev set up as a route?

**Reply by ankitjaggi** [reply]
Yes. All records have been added based on the documentation. Not sure if I am missing something.

**Comment by sinnbeck** [comment]
What type of route is it? Smtp, http or address ?

**Comment by ankitjaggi** [comment]
I forgot to add a route. 🤦
